eng/pipelines/linux.yml

```diff
@@ -106,12 +106,21 @@ jobs:
                 _dockerContainer: alpine_36_container
                 _buildScriptPrefix: ''
                 _buildExtraArguments: /p:RuntimeOS=linux-musl
+            
+            ${{ if eq(parameters.isOfficialBuild, 'true') }}:
+              musl_arm64_Release:
+                _BuildConfig: Release
+                _architecture: arm64
+                _framework: netcoreapp
+                _dockerContainer: alpine_37_arm64_container
+                _buildScriptPrefix: 'ROOTFS_DIR=/crossrootfs/arm64 '
+                _buildExtraArguments: -warnAsError false /p:BuildNativeClang=--clang5.0 /p:RuntimeOS=linux-musl
```

Comment:
Should RuntimeOS be linux-musl-arm64?

Reply:
Nope, the RuntimeOS doesn't include arch in corefx, then we calculate the RID using `ArchGroup`.

Look at the build output how it is generated correctly:

`Verifying closure of runtime.linux-musl-arm64.Microsoft.Private.CoreFx.NETCoreApp runtime assemblies
  Verifying no duplicate types in runtime.linux-musl-arm64.Microsoft.Private.CoreFx.NETCoreApp runtime assemblies
  Microsoft.Private.CoreFx.NETCoreApp -> /__w/1/s/artifacts/packages/Release/specs/runtime.linux-musl-arm64.Microsoft.Private.CoreFx.NETCoreApp.nuspec`